examples/step-74/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+##
+#  CMake script for the step-12 tutorial program:
+##
+
+# Set the name of the project and target:
+SET(TARGET "step-laplace-FEIV")
```

Comment:
I think you will want to rename the target to step-74 instead of step-laplace-FEIV